addons/website/static/src/builder/plugins/customize_website_plugin.js

```diff
@@ -127,6 +127,9 @@ export class CustomizeWebsitePlugin extends Plugin {
         if (reloadBundles) {
             await this.reloadBundles();
         }
+        // since we do not reload on website variables customization
+        // we need to trigger resize to have navbar layout recomputed
+        this.window.dispatchEvent(new Event("resize"));
```

Comment:
Maybe don't do it for all customizeWebsiteVariables. Maybe we should add a custom action that extends customizeWebsiteVariable. For your specific use case (Page Layout)